docs/cimpl6LanguageReference.md

```diff
@@ -78,70 +78,52 @@ Any periods in the _namespace_ should be replaced by dashes. For example, for a
 
-* Configuration files: _ig-igname-config.json_
-* Content profile files: _ig-igname-cp.json_
-* IG examples: _ignameExampleName.json_
-* IG HTML pages: _anyName.html_
+* Configuration files: _ig-**myigname**-config.json_
+* Content profile files: _ig-**myigname**-cp.json_
+* IG examples: _igname-**myexamplename**.json_
+* IG HTML pages: _**anyName**.html_
 
 Detail about these files and how they are used to compile CIMPL models and create FHIR IGs can be found in the [CIMPL Tooling Reference Manual](#cimpl6ToolingReference.md).
 
 ### Namespace Names
 
-Best practice is to follow the naming convention pattern of _organization.domain_ or _project.domain_ (followed by subdomains if necessary).
+Best practice is to follow the naming convention pattern _project_ or _project.domain_ (followed by subdomains if necessary).
 
 | Example | Purpose |
 |-------|-------|
-| _obf.lab_ | Laboratory profiles belonging to Objective FHIR |
-| _odh_ | Occupational Data for Health |
-| _onco.core_ | Core namespace for oncology-related profiles |
+| `obf.lab` | Laboratory profiles belonging to Objective FHIR |
+| `odh` | Occupational Data for Health |
+| `onco.core` | Core oncology profiles |
 
-### Class Names
+>**Note:** While it is best practice to use only lower case letters, the strict requirements only require the word to begin with a lowercase letter. The rest of the namespace allows for mixed casing and hyphens.
 
-CIMPL classes are conventionally defined in [PascalCase](http://wiki.c2.com/?PascalCase). For example:
+### Class Names
 
-```
-Dosage
-DetectionTime
-AdverseEvent
-Neutrophils100WBCNFrPtBldQnAutoCntLabObs
-```
+CIMPL classes are conventionally defined in [PascalCase](http://wiki.c2.com/?PascalCase), for example, `Dosage`, `DetectionTime`, and `Neutrophils100WBCNFrPtBldQnAutoCntLabObs`.
 
 >**Note:** Letters, numbers, and hyphens are allowed. However, class names must begin with an uppercase letter.
 
 ### Fully Qualified Names
 
-CIMPL does not support duplicate class names within a single namespace. However, the same name may occur in different namespaces. In the case of naming collisions due to an included namespace in [`Uses`](#Uses), you have to refer to the class by its fully qualified name (FQN). The FQN is a combination of the `Element` namespace and its declared name (through concatenation delimited by a period). For example, the `Element` _CancerCondition_ in the namespace _onco.core_ has the following FQN:
-
-```
-onco.core.CancerCondition
-```
+CIMPL does not support duplicate class names within a single namespace. However, the same name may occur in different namespaces. In the case of naming collisions due to an namespace included by a [`Uses`](#Uses) statement, you have to refer to the class by its fully qualified name (FQN). The FQN is a concatenation of the namespace name and the class name (delimited by a period). For example, the class `CancerCondition` in the namespace `onco.core` has the FQN `onco.core.CancerCondition`. 
```

Comment:
See line 72 for reference to onco.core - italics vs: code block?  Not clear when to use one or the other.

Reply:
onco.core.CancerCondition is a quote from a CIMPL file, so code is correct.